media/css/firefox/new/private-not-option.less

```diff
@@ -0,0 +1,74 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../sandstone/lib.less';
+
+// generates a < tablet media query for all sections (except intro)
+.common-mobile-props(@padding: 250px) {
+    @media only screen and (max-width: @breakTablet) {
+        background-position: bottom 20px center;
+        min-height: 0;
+        padding-bottom: @padding;
+
+        .copy {
+            float: none;
+            width: auto;
+        }
+    }
+}
+
+#intro {
+    background-image: url('/media/img/firefox/new/private-not-option/intro-high-res.png');
+
+    background-position: right top;
+    background-repeat: no-repeat;
+
+    @media only screen and (max-width: @breakDesktop) {
+        background-image: url('/media/img/firefox/new/private-not-option/intro.png');
+
+        p {
+            .font-size(22px);
+        }
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        p {
+            .font-size(18px);
+            line-height: 1.6;
+            margin-bottom: 40px;
+
+            br {
+                display: none;
+            }
+        }
+
+        .content {
+            text-align: center;
+        }
+    }
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        background-image: url('/media/img/firefox/new/private-not-option/intro-mobile.png');
+    }
+}
+
+#content-1 .content {
+    .at2x('/media/img/firefox/new/private-not-option/private-browsing.png', 256px, 266px);
+
+    @media only screen and (max-width: @breakDesktop) {
+        .background-size(300px 182px);
+    }
+}
+
+#content-2 .content {
+    .at2x('/media/img/firefox/new/private-not-option/private-mobile.png', 400px, 317px);
+}
+
+#content-3 .content {
+    .at2x('/media/img/firefox/new/private-not-option/forget-button.png', 400px, 243px);
+
+    @media only screen and (max-width: @breakDesktop) {
+        .background-size(300px 182px);
+    }
+}
```

Comment:
I don't see any of the three `.content` selectors above being in use. Can they be deleted?